airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -223,58 +223,25 @@ def _read_incremental(
         stream_name = configured_stream.stream.name
         stream_state = state_manager.get_stream_state(stream_name, stream_instance.namespace)
 
-        if stream_state and "state" in dir(stream_instance):
+        if stream_state and "state" in dir(self):
```

Comment:
I'm a little surprised by this change since your not adding `state` to the AbstractSource. How does this work?

Reply:
I'm dumb and it's copy/paste from when I moved this to the stream. I decided to move this back to AbstractSource. Hence, this should be `dir(stream_instance)`